dotCMS/src/integration-test/java/com/dotcms/experiments/business/ExperimentAPIImpIT.java

```diff
@@ -159,4 +162,71 @@ public void testStartExperiment_shouldPublishContent()
         }
 
     }
+
+    /**
+     * Method to test: {@link ExperimentsAPI#addVariant(String, String, User)} (String, User)}
+     * When: The {@link com.dotcms.experiments.model.AbstractExperimentVariant#ORIGINAL_VARIANT} variant is created
+     * Should: copy the page and the contentlet related to the page in the new variant
+     */
+    @Test
+    public void testAddOriginalVariant_shouldCopyPageAndContentletToVariant()
+            throws DotDataException, DotSecurityException {
+        final HTMLPageAsset page = APILocator.getHTMLPageAssetAPI().fromContentlet(
+                TestDataUtils.getPageContent(true, 1));
+        HTMLPageDataGen.publish(page);
+
+        final Container container = new ContainerDataGen().nextPersisted();
+
+        final ContentType contentType = new ContentTypeDataGen().nextPersisted();
+
+        final Contentlet content1 = new ContentletDataGen(contentType)
+                .nextPersisted();
+        ContentletDataGen.publish(content1);
+
+        final Contentlet content2 = new ContentletDataGen(contentType)
+                .nextPersisted();
+        ContentletDataGen.publish(content2);
+
+        final Contentlet content3 = new ContentletDataGen(contentType)
+                .nextPersisted();
+        ContentletDataGen.publish(content3);
+
+        new MultiTreeDataGen().setPage(page).setContainer(container)
+                .setContentlet(content1).nextPersisted();
+
+        new MultiTreeDataGen().setPage(page).setContainer(container)
+                .setContentlet(content2).nextPersisted();
+
+        new MultiTreeDataGen().setPage(page).setContainer(container)
+                .setContentlet(content3).nextPersisted();
+
+        final Experiment newExperiment = new ExperimentDataGen()
+                .page(page)
+                .addVariant("Test Gray Button")
+                .nextPersisted();
+
+        final ExperimentVariant originalVariant = newExperiment.
+                trafficProportion().variants().first();
+
+        List<Contentlet> experimentContentlets = APILocator.getContentletAPI()
+                .getAllContentByVariants(APILocator.systemUser(),
+                        false, originalVariant.id());
+
+        // expecting the page + the 3 contentlets
+        assertEquals(4, experimentContentlets.size());
```

Comment:
maybe we should assert something else to make sure that the right contentlet was created